drivers/storage/ec2/ec2.go

```diff
@@ -921,3 +1022,17 @@ func configRegistration() *gofig.Registration {
 	r.Key(gofig.String, "", "", "", "aws.region")
 	return r
 }
+
+func awsErrCode(err error) string {
```

Comment:
Yes, its unused in this version. I was using it to parse error code in `createVolumeCreateVolume` function, but removed that call in final version.
And, while its currently unused, I think it can be useful in further development.
But if its suggested, I will remove it for clearer code.